library/test/src/types.rs

```diff
@@ -79,10 +79,10 @@ impl fmt::Display for TestName {
 // may need to come up with a more clever definition of test in order
```

Comment:
We should update this documentation as well to note that both panicking and returning Err(...) are valid ways to fail a test.

Reply:
Done.